Core.UnitTests/Implementation/ReflectionServiceTest.cs

```diff
@@ -168,24 +168,25 @@ public void InstantiateAssembledType_WithExactAssembledType ()
       var arguments = ParamList.Create ("abc", 7);
       var allowNonPublic = BooleanObjectMother.GetRandomBoolean();
       _constructorForAssembledTypeCacheMock
-          .Expect (mock => mock.GetOrCreateConstructorCall (assembledType, arguments.FuncType, allowNonPublic))
-          .Return (new Func<string, int, object> ((s, i) => "blub"));
+          .Setup (mock => mock.GetOrCreateConstructorCall (assembledType, arguments.FuncType, allowNonPublic))
+          .Returns (new Func<string, int, object> ((s, i) => "blub"))
+          .Verifiable();
 
       var result = _service.InstantiateAssembledType (assembledType, arguments, allowNonPublic);
 
-      _typeCacheMock.VerifyAllExpectations();
+      _typeCacheMock.Verify();
       Assert.That (result, Is.EqualTo ("blub"));
     }
     
     [Test]
     public void PrepareAssembledTypeInstance_Initializable ()
     {
-      var initializableObjectMock = MockRepository.GenerateMock<IInitializableObject>();
+      var initializableObjectMock = new Mock<IInitializableObject>();
       var reason = BooleanObjectMother.GetRandomBoolean() ? InitializationSemantics.Construction : InitializationSemantics.Deserialization;
 
-      _service.PrepareExternalUninitializedObject (initializableObjectMock, reason);
+      _service.PrepareExternalUninitializedObject (initializableObjectMock.Object, reason);
 
-      initializableObjectMock.AssertWasCalled (mock => mock.Initialize (reason));
+      initializableObjectMock.Verify (mock => mock.Initialize (reason), Times.AtLeastOnce());
```

Comment:
Times(1)
Just do a find-all on AtLeastOnce, probably will want to replace them all with Times(1)